progs/jubjub.py

```diff
@@ -219,3 +220,29 @@ def __str__(self):
 
     async def double(self):
         return self
+
+
+async def share_mul(context: Mpc, p: Point, x: list) -> SharedPoint:
+    """
+    The multiplication of the share of a field element and a point
+    e.g. [x]G -> [X], where G is a point on the given elliptic curve
+    x is the bitwise shared value,
+    starting from the most significant bit.
+
+    Source: Section `Scalar multiplication` of
+    https://andrea.corbellini.name/2015/05/17/elliptic-curve-cryptography-a-gentle-introduction/
```

Comment:
You'll probably want to add in `# noqa: E501` at the end of this line to pass the linter